### ISSUE TITLE
invalid option: --control-url

### ISSUE THREAD

**Issue by sigmike** [puma/puma#1477]
### Steps to reproduce

1) Install puma 3.11.0

2) Create a config with only `activate_control_app 'unix:///tmp/puma.sock'`

3) Run `pumactl -F <config file> start`

### Expected behavior

Starting an complaining about application not configured.

### Actual behavior

```
invalid option: --control-url
/home/mike/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/puma-3.11.0/lib/puma/cli.rb:47:in `initialize'
/home/mike/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/puma-3.11.0/lib/puma/control_cli.rb:258:in `new'
/home/mike/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/puma-3.11.0/lib/puma/control_cli.rb:258:in `start'
/home/mike/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/puma-3.11.0/lib/puma/control_cli.rb:223:in `run'
/home/mike/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/puma-3.11.0/bin/pumactl:8:in `<top (required)>'
/home/mike/.rbenv/versions/2.4.2/bin/pumactl:23:in `load'
/home/mike/.rbenv/versions/2.4.2/bin/pumactl:23:in `<main>'
```

### System configuration

**Ruby version**: 2.4.2
**Rails version**: N/A
**Puma version**: 3.11.0

### Comments

Seems to be caused by commit https://github.com/puma/puma/commit/459ab196369be9355432d43097e43a5e0761b8e9

Puma 3.10.0 works fine.

**Comment by crevete** [comment]
I've got the same issue and go back to 3.10 as workaround for now.

**Comment by jlduran** [comment]
https://github.com/puma/puma/commit/459ab196369be9355432d43097e43a5e0761b8e9

**Comment by jxa** [comment]
I get the same. The problem stems from an option mismatch between pumactl and puma cli. On puma cli the option is `--control` and in pumactl the option is called `control-url`. 

Putting together a PR as soon as I figure out how to write a test for this.

**Comment by chickenham** [comment]
I noticed that a new version of puma was released but this is still not fixed.  Is this going to be fixed at some point - or is there a workaround?

**Comment by crevete** [comment]
I don't think there is a workaround... strange that nobody try to fix this critical bug from 12/2017...

**Comment by crevete** [comment]
Why you don't fix this critical bug in maintenance releases 3.11.x?

**Comment by jdoss** [comment]
This is a blocker on us upgrading Puma as well.

**Comment by jxa** [comment]
https://github.com/puma/puma/pull/1487 is a fix, but I need some help getting the new test to pass on CI. If anyone has advice or the time to look into it I'd appreciate the help.

**Comment by crevete** [comment]
@nateberkopec Why did you delete my comment ? Can you please explain me whe you continue to release new versions without fixing this blocker issue ?

**Comment by frankpinto** [comment]
Just ran into this myself while upgrading from Upstart in Puma 14.04 to 16.04 and wanting to use pumactl start as the ExecStart. Any updates?

**Comment by jxa** [comment]
The fix is merged to master but not in time for the latest release. It would be really swell if they did a bugfix release for this issue.

What do you think @schneems  ?

**Comment by schneems** [comment]
I have another metric I want to expose via Puma.stats probably next week. I’m fine if this goes out before then though.

**Comment by schneems** [comment]
I cut a release Can his VR closed?

**Comment by sigmike** [comment]
Yes it's fixed in 3.12.0. Thank you.
